app/src/main/java/com/globant/practice/domain/model/Profile.java

```diff
@@ -0,0 +1,68 @@
+package com.globant.practice.domain.model;
+
+/**
+ * Represents the user profile.
+ * Created by jonathan.vargas on 5/04/2017.
+ */
+
+public final class Profile {
+    private final String login;
+    private final String avatarUrl;
```

Comment:
You need serialized name in this attribute since the json attribute name is "**avatar_url**". Check other cases.